app/lib/search/updater.dart

```diff
@@ -229,6 +229,19 @@ class _PeriodicUpdateTaskSource implements TaskSource {
 
   @override
   Stream<Task> startStreaming() async* {
+    // Scan for potential derived tag updates at startup.
+    // If the package has known platform tags (previous analysis was successful),
+    // but has no sdk: tags, we'll schedule it to be updated.
+    // TODO: remove this code after the current release becomes stable
+    final missingTagTasks = _snapshot.documents.values
+        .where((pd) =>
+            (pd.platforms != null && pd.platforms.isNotEmpty) &&
+            (pd.tags == null || pd.tags.every((t) => !t.startsWith('sdk:'))))
```

Comment:
Packages that fail analysis will get updated all the time..

Reply:
Conditioning on  `(pd.platforms != null && pd.platforms.isNotEmpty)` means we only try this with packages that were not failing the analysis.